gadopt/diagnostics.py

```diff
@@ -11,20 +14,21 @@ class GeodynamicalDiagnostics:
       u:         Firedrake function for the velocity
       p:         Firedrake function for the pressure
       T:         Firedrake function for the temperature
-      bottom_id: bottom boundary identifier.
-      top_id:    top boundary identifier.
-      degree:    degree of the polynomial approximation.
+      bottom_id: bottom boundary identifier
+      top_id:    top boundary identifier
+      diag_vars: dictionary containing additional diagnostic-related functions
```

Comment:
This is very vague: the user will have no idea what to put here.

Reply:
Agreed I should document it better. Note that it depends on what is decided below.